src/cpp/CMakeLists.txt

```diff
@@ -30,13 +30,16 @@ function(ov_genai_build_jinja2cpp)
         set(JINJA2CPP_PIC ON CACHE BOOL "")
 
         add_subdirectory("${jinja2cpp_SOURCE_DIR}" "${jinja2cpp_BINARY_DIR}" EXCLUDE_FROM_ALL)
+        # openvino::runtime exports _GLIBCXX_USE_CXX11_ABI=0 on CenOS7.
+        # It needs to be propagated to every lib GenAI links with. It's
+        # enough to propagate it to fmt, because fmt propagates to
+        # jinja2cpp.
+        target_link_libraries(fmt PUBLIC openvino::runtime)
```

Comment:
good catch
looks like `target_compile_defitions(fmt PUBLIC $<TARGET_PROPERTY:openvino::runtime,INTERFACE_COMPILE_DEFINITIONS>)` is enough